react/features/base/tracks/middleware.web.ts

```diff
@@ -20,16 +24,20 @@ import {
     TRACK_UPDATED
 } from './actionTypes';
 import {
+    createLocalTracksA,
     showNoDataFromSourceVideoError,
     toggleScreensharing,
+    trackMuteUnmuteFailed,
     trackNoDataFromSourceNotificationInfoChanged
 } from './actions.web';
 import {
-    getTrackByJitsiTrack, logTracksForParticipant
+    getLocalJitsiAudioTrackSettings,
+    getTrackByJitsiTrack, isUserInteractionRequiredForUnmute, logTracksForParticipant,
+    setTrackMuted
 } from './functions.web';
-import { ITrack } from './types';
+import { _getLocalTrack } from './middleware.any';
```

Comment:
you can use `getLocalTrack` from `./functions` directly and make sure that the `includePending` param is set to true